backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -558,6 +564,20 @@ export class UserResolver {
     return true
   }
 
+  @Authorized([RIGHTS.QUERY_OPT_IN])
+  @Query(() => Boolean)
+  async queryOptIn(@Arg('optIn') optIn: string): Promise<boolean> {
+    const optInCode = await LoginEmailOptIn.findOneOrFail({ verificationCode: optIn })
+    // Code is only valid for 10minutes
+    const timeElapsed = Date.now() - new Date(optInCode.updatedAt).getTime()
+    if (timeElapsed > parseInt(CONFIG.EMAIL_CODE_VALID_TIME.toString()) * 60 * 1000) {
```

Comment:
we might need to separate regenerate time and valid time:
- every 10 minutes i can request a new code
- code is valid for 24hs
But this is not part of this PR - i am fine with this!

Reply:
Ah OK then I didn't understand it right will put this in a follow ticket

Reply:
thats a new thing related to https://github.com/gradido/gradido/issues/1566 and not part of this ticket.